app/views/admin/claims/index.html.erb

```diff
@@ -31,7 +33,7 @@
               <th scope="col" class="govuk-table__header">Decision warning</th>
             <% end %>
             <th scope="col" class="govuk-table__header">Decision deadline</th>
-            <th scope="col" class="govuk-table__header"></th>
+            <td scope="col" class="govuk-table__header"></td>
```

Comment:
I don't think a `td` can have a `scope` attribute in HTML5. Maybe we should either remove the scope or add an `aria-label` that describes the column?

Reply:
Thanks bud, I've added an aria role as suggested 👍

Reply:
I think `td[role=colheader]` is semantically equal to `th[scope=col]` - so I think the issue of having an empty column header still stands? I could be wrong, I'm not sure how screen readers will read tables. My guess would be that each column needs a header for when you tab through the cells?